applications/MeshMovingApplication/python_scripts/mesh_solver_base.py

```diff
@@ -31,12 +31,17 @@ class MeshSolverBase(object):
     settings -- Kratos parameters for general mesh motion settings.
     mesh_model_part -- the mesh motion model part.
     """
-    def __init__(self, mesh_model_part, custom_settings):
+    def __init__(self, model, custom_settings):
+        super(MeshSolverBase,self).__init__(model, custom_settings)
+
         default_settings = KratosMultiphysics.Parameters("""
         {
-            "solver_type" : "mesh_solver_structural_similarity",
-            "buffer_size": 3,
-            "echo_level": 0,
+            "solver_type"           : "mesh_solver_structural_similarity",
+            "buffer_size"           : 3,
```

Comment:
do we actually need the buffer_size to be 3? i would say no, depending on how we compute the mesh velocity.

the bottomline is that instead of telling the buffer size we should tell which kind of time integration algorithm are we assuming in the computation of mesh velocity.

Reply:
it does, we are currently working on this

In order to keep the PRs small I postponed it to the next PR though